Add RSS feed with basic spec updates

DanielOaks opened the pull request:
For some time, people have wanted a way to just subscribe for the occasional update on what we're doing. This PR introduces an RSS feed to let people subscribe for the occasional update on what we're doing, the status of our specs, and recent changes.

This also fills out the WG page a fair bit, which is good. Solves #83 and #103.

Example post:
<img width="1349" alt="screen shot 2016-11-20 at 12 35 41 pm" src="https://cloud.githubusercontent.com/assets/251281/20459903/4e814a58-af1e-11e6-8320-0cf2a90d2663.png">

And how it's displayed on the WG page:
<img width="1350" alt="screen shot 2016-11-20 at 12 35 27 pm" src="https://cloud.githubusercontent.com/assets/251281/20459904/54d32264-af1e-11e6-8eac-0f003bb69ac4.png">

jwheare commented:
Great! Let's get a link to the roadmap milestone on the WG page while we're at it. That and status updates/changelog should probably be on the home page too.

DanielOaks commented:
Yeah threw roadmap link on the WG as 'the top article', should go well. I'm iffy on throwing it on the home page too, just due to how much we already have on there and having to modify this a decent amount to get it to play nicely there. Mind if we leave that 'til after this is merged/in, since it may take a bit of time to play with and see exactly how we wanna tackle that?

dequis commented on wg.html:
```diff
@@ -33,3 +33,32 @@
+      <div class="post">
+          <h3><a href="https://github.com/ircv3/ircv3-specifications/milestone/4">Roadmap</a></h3>
+          <div class="post-content">
+              <p>Our roadmap <a href="https://github.com/ircv3/ircv3-specifications/milestone/4"><sup>[link]</sup></a> always contains info on our direction moving forward, the sort of features we have planned and our priorities.</p>
```
`roadmap <a><sup>[link]</sup></a>` -> `<a>roadmap</a>`?

jwheare commented:
Yeah fine by me to delay home page stuff.

DanielOaks commented:
Cool, cleaned up the language there and added notes for STS and SNI.

<img width="1342" alt="screen shot 2016-11-20 at 2 00 37 pm" src="https://cloud.githubusercontent.com/assets/251281/20460275/d7c0bece-af29-11e6-886e-3ade0fd6e1ed.png">

Should be good to go with this?

ChickenNuggers commented:
Is there a reason links are not implemented using just anchor tags for `[link]`s?

DanielOaks commented:
Because they're a bit difficult to see and this brings them out a bit more. I'll make links in general easier to see but I'd probably prefer having the external links like this (particularly as this is kinda like a reference page, having the explicit external links could make sense here).

ChickenNuggers commented:
> having the explicit external links could make sense here

Now that you mention it, that's a good point.

jwheare commented:
LGTM
